AutoMerge.jl/test/runtests.jl

```diff
@@ -0,0 +1,54 @@
+using Dates
+using GitHub
+using JSON
+using Pkg
+using Printf
+using AutoMerge
+using RegistryCI
+using Test
+using TimeZones
+using ReferenceTests
+
+# Starting with Julia 1.7, when you use the Pkg server registry, the registry tarball does
+# not get unpacked, and thus the registry files are not available. Of course, AutoMerge
+# requires that the registry files are available. So for the AutoMerge test suite, we will
+# disable the Pkg server.
+ENV["JULIA_PKG_SERVER"] = ""
+
+@static if v"1.6-" <= Base.VERSION < v"1.11-"
+    # BrokenRecord fails to precompile on Julia 1.11
+    let
+        # The use of `VersionNumber`s here (e.g. `version = v"foo.bar.baz"`) tells Pkg to install the exact version.
+        brokenrecord = Pkg.PackageSpec(name = "BrokenRecord", uuid = "bdd55f5b-6e67-4da1-a080-6086e55655a0", version = v"0.1.9")
+        jld2 = Pkg.PackageSpec(name = "JLD2", uuid = "033835bb-8acc-5ee8-8aae-3f567f8a3819", version = v"0.4.33")
+        pkgs = [brokenrecord, jld2]
+        Pkg.add(pkgs)
+    end
+    import BrokenRecord
+end
+
+@testset "AutoMerge.jl" begin
+    @testset "TagBot.jl unit tests" begin
+        # if v"1.0" <= VERSION < VersionNumber(1, 5, typemax(UInt32))
+        if false
+            @info("Running the TagBot.jl unit tests", VERSION)
+            include("tagbot-unit.jl")
+        else
+            @warn("Skipping the TagBot.jl unit tests", VERSION)
+        end
+    end
+
+    @testset "AutoMerge.jl unit tests" begin
+        @info("Running the AutoMerge.jl unit tests")
+        include("automerge-unit.jl")
+    end
+
+    AUTOMERGE_RUN_INTEGRATION_TESTS =
+        get(ENV, "AUTOMERGE_RUN_INTEGRATION_TESTS", "")::String
+    if AUTOMERGE_RUN_INTEGRATION_TESTS == "true"
+        @testset "AutoMerge.jl integration tests" begin
+            @info("Running the AutoMerge.jl integration tests")
+            include("automerge-integration.jl")
+        end
+    end
+end
```

Comment:
Can you make sure that all files end in a trailing newline character (for compliance with the POSIX standard)?